src/main/java/oak/controller/TerminalController.java

```diff
@@ -0,0 +1,53 @@
+package oak.controller;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import oak.exceptions.InvalidInputException;
+import oak.feedback.FeedbackService;
+import oak.feedback.enums.CommandEnum;
+
+/**
+ * The type Terminal controller, handles the listening of messages and printing of responses to the terminal
+ */
+public class TerminalController {
+    private FeedbackService feedbackService = new FeedbackService();
+
+    /**
+     * Print welcome message and start listening for user inputs
+     */
+    public void start() {
+        System.out.println(this.feedbackService.getWelcomeMessage());
+        this.listen();
+    }
+
+    /**
+     * Start listening for user inputs and prints out the messages obtained from FeedbackService
+     * Exits when the user inputs 'bye'
+     */
+    private void listen() {
+        Scanner scanner = new Scanner(System.in);
+        boolean stop = false;
```

Comment:
I guess it might be better to use another name like 'isStop'? (referred to "Boolean variables/methods should be named to sound like booleans" in Java coding standard) :))